src/EnergyPlus/Autosizing/Base.cc

```diff
@@ -413,6 +413,9 @@ void BaseSizer::selectSizerOutput(EnergyPlusData &state, bool &errorsFound)
                 }
             }
             if (!this->wasAutoSized) this->autoSizedValue = this->originalValue;
+        } else if (this->wasAutoSized && this->autoSizedValue != DataSizing::AutoSize) {
+            this->reportSizerOutput(
+                state, this->compType, this->compName, "Design Size " + this->sizingStringScalable + this->sizingString, this->autoSizedValue);
```

Comment:
This opens up the possibility that an autosized value can be negative. I find it 1) hard to believe we haven't seen this before and 2) whether it is smart to allow a negative number for all autosizable fields or just some (e.g., temperature).

I guess since the autosized value was "something" other than DataSizing::AutoSize then the sizing routine did do something and should be reported.